django/db/backends/sqlite3/features.py

```diff
@@ -42,6 +42,9 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_json_field_contains = False
     supports_update_conflicts = Database.sqlite_version_info >= (3, 24, 0)
     supports_update_conflicts_with_target = supports_update_conflicts
+    supports_generated_field = True
+    supports_generated_field_params = False
+    supports_generated_field_virtual = True
```

Comment:
Is it true for all supported versions of SQLite (i.e. 3.21+)? I tried with SQLite 3.26 and tests crash:
```
...
django.db.utils.OperationalError: near "AS": syntax error
```

Reply:
As I already reported in the developer mailing list [1], SQLite added support for Generated Columns in version 3.31.0 (2020-01-22) [2].

- [1] https://groups.google.com/g/django-developers/c/9Mf7YqDA4bg/m/D5a5jUCACwAJ
- [2] https://www.sqlite.org/releaselog/3_31_0.html